src/genomic_tools_lib/external_tools/dap/RunDAP.py

```diff
@@ -7,16 +7,43 @@
 import logging
 import traceback
 from collections import namedtuple
-from subprocess import call
+from subprocess import call, PIPE
 
 from ...file_formats import SBAM
 from ...individual_data.Utilities import _StudyBasedContext
 from ...individual_data import Utilities as StudyUtilities
 from ...Exceptions import ReportableException
 from ... import Utilities
 
+COMMAND_WITH_PRIORS = """#!/usr/bin/env bash
+
+unset OMP_NUM_THREADS
```

Comment:
Although I think this is ok, it is specific to how `dap` was built. MAybe put a note somewhere?